src/main/java/dev/group4/repos/UserRepo.java

```diff
@@ -10,4 +10,6 @@
 @Repository
 public interface UserRepo  extends JpaRepository<User,String> {
     //TODO add custom method signatures
+    User findUserByUsernameAndPassword(String username, String password);
+
```

Comment:
When I wrote this in p0 I just did a find by username and compared the password of the user to return either the user or null in the service layer depending if it was right. If this ends up having issues it might be easier to address in the services.